templates/index.html

```diff
@@ -7,12 +7,32 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+ 
+
   <div class="row">
+    <div class="col-md-12">
+      <form action="/newitems" method="post">
+      <h2>New Items</h2>
+        <label for="name">Title:</label>
+        <input type="text" id="title_field" name="Title">
+        <button type="submit">Submit</button>
+    </form>
```

Comment:
The `h2` and `</form>` lines should each be indented one more level to accurately match the nesting of the elements.